lib/network/ci-external-load-balancer.ts

```diff
@@ -42,7 +42,7 @@ export class JenkinsExternalLoadBalancer {
     const accessPort = props.useSsl ? 443 : 80;
 
     this.listener = this.loadBalancer.addListener('JenkinsListener', {
-      sslPolicy: props.useSsl ? SslPolicy.TLS12 : undefined,
+      sslPolicy: props.useSsl ? SslPolicy.RECOMMENDED : undefined,
```

Comment:
Lets use a fixed tls policy such as 2017/2019, as recommended version tends to change overtime.

Reply:
Hey @peterzhuamazon as per the [document](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html#describe-ssl-policies) its better we go with the recommend one.
`We recommend the ELBSecurityPolicy-2016-08 policy for compatibility.`